packages/common/src/ui/components/modals/AlertModal/AlertModal.tsx

```diff
@@ -1,44 +1,38 @@
-import React from 'react';
-import { Grid, Typography } from '@mui/material';
-import { DialogButton } from '../../buttons';
-import { AlertIcon } from '@common/icons';
-import { BasicModal } from '@common/components';
-
+import React, { useContext } from 'react';
+import { AlertModalContext } from './AlertModalContext';
 export interface AlertModalProps {
+  important?: boolean;
   message: React.ReactNode;
   open: boolean;
   onOk: () => void;
   title: string;
 }
 
+// This is really just a convenience component
+// allowing you to use the modal in a declarative syntax
+// without creating multiple overlaying modals
+// Set the important prop only if this is a critical alert
+// which should only be superceded by other critical alerts
+// Apart from the important / non-important distinction,
+// the latest caller wins, and will be displayed
```

Comment:
Looks like these changes are from a previous PR, but they're not in `#1098-remove-login-store` which this branch is based off.

Reply:
I just merged in that branch when testing.. since there was a problem getting the stores and I couldn't log in!
Will tidy up.